src/main/kotlin/solve/importer/view/DirectoryPathView.kt

```diff
@@ -19,55 +22,80 @@ class DirectoryPathView : View() {
+    private val roboto = Style.fontCondensed
 
-    private val directoryLabel = label("Project directory") {
-        maxWidth = 300.0
-        textOverrun = OverrunStyle.ELLIPSIS
-        controller.directoryPath.onChange {
-            tooltip(controller.directoryPath.value)
-            val x = controller.directoryPath.value
-            this.text = if (x == null) "Project directory"
-            else "Project directory\n$x"
+    private val directoryField = mfxTextField {
+        isEditable = false
+        isAllowEdit = false
+
+        enableWhen {
+            controller.projectAfterPartialParsing.isNotNull
         }
-    }
+        textFill = Color.valueOf(Style.onBackgroundColor)
 
-    private val changeButton = button("Change") {
-        action {
-            val dir = directoryChooser.showDialog(currentStage)
-            controller.directoryPath.set(dir?.absolutePath)
+        prefHeight=48.0
+        prefWidth=280.0
+        BorderPane.setAlignment(this, Pos.CENTER)
+        BorderPane.setMargin(this, Insets(16.0, 0.0,0.0, 0.0))
+
+        style = "-fx-border-color: #${Style.primaryColorLight}; -fx-font-size: 15px;"
+
+        floatMode = FloatMode.BORDER
+        font = Font.font(roboto, 14.0)
+
+        floatingText = "Select project directory"
+        controller.directoryPath.onChange {
+            floatingText = if (it != null && it != ""){
+                "Selected project directory"
+            } else{
+                "Select project directory"
+            }
         }
-    }
 
-    private val countImagesLabel = label {
-        visibleWhen { controller.projectAfterPartialParsing.isNotNull }
         controller.projectAfterPartialParsing.onChange {
-            val countFiles = it?.projectFrames?.count()
-            this.text = "$countFiles images found"
-            this.graphic = ImageView(filesCountIcon)
+            if (controller.projectAfterPartialParsing.value != null){
+                tooltip(controller.directoryPath.value).apply {
+                    font = Font.font(Style.font, 12.0)
+                    style = "-fx-background-color: #${Style.surfaceColor}; -fx-text-fill: #707070;"
+                }
+            }
+            text = if (it != null){
+                controller.directoryPath.value
+            } else{
+                ""
+            }
         }
     }
 
-    private val countErrorsLabel = label {
-        visibleWhen { controller.projectAfterPartialParsing.isNotNull }
-        padding = Insets(3.0, 0.0, 5.0, 0.0)
-        controller.projectAfterPartialParsing.onChange {
-            var countErrors = 0
-            it?.projectFrames?.forEach { frame ->
-                if (frame.image.errors.isNotEmpty())
-                    countErrors += 1
-            }
-            this.text = "$countErrors errors"
-            this.graphic = ImageView(errorsCountIcon)
+    private val labelName = label("Import a directory") {
```

Comment:
Text should not be trimmed
![image](https://user-images.githubusercontent.com/55746901/232252088-7fdd7630-9ff6-4f3e-9f29-e45cebf7b554.png)